### ISSUE TITLE
Adjust Microphone and In-App Audio Levels

### ISSUE THREAD

**Issue by noho501** [pedroSG94/RootEncoder#1525]
Hi @pedroSG94  

Could you please provide sample code to increase or decrease the microphone and in-app audio levels while screen recording?

Thank you.

**Comment by pedroSG94** [comment]
I don't understand you. What do you means with audio levels?
Volume? The volume should be handle in player side using the volume control of the device.

**Comment by noho501** [comment]
Yes, volume. How do you control the microphone volume? It should function like the image below.

![IMG_1312](https://github.com/user-attachments/assets/4cd66620-d29f-4fe5-af06-702160ce2b1c)

**Comment by pedroSG94** [comment]
Hello,

Android doesn't provide a way to control microphone volume. The volume should be control in player side, as I said you, using the button in the device to increase or reduce volume
